corehq/apps/app_manager/xform.py

```diff
@@ -543,7 +545,11 @@ def validate_xform(source):
         source = source.encode("utf-8")
     # normalize and strip comments
     source = ET.tostring(parse_xml(source))
-    validation_results = formtranslate.api.validate(source)
+    try:
+        validation_results = nimbus_api.validate_form(source)
+    except NimbusAPIException:
+        return
```

Comment:
In the case of an error I thought the best thing to do is just ignore it since there is nothing the user can do about it. This does mean that could create a build with an invalid form. The alternative is to give a message saying that we couldn't validate their form but this also prevents them from doing any case management and some other things.